kornia/augmentation/_2d/intensity/gamma.py

```diff
@@ -0,0 +1,83 @@
+import warnings
+from typing import Any, Dict, List, Optional, Tuple, Union, cast
+
+from torch import Tensor
+
+from kornia.augmentation import random_generator as rg
+from kornia.augmentation._2d.intensity.base import IntensityAugmentationBase2D
+from kornia.enhance.adjust import adjust_gamma
+
+
+class RandomGamma(IntensityAugmentationBase2D):
+    r"""Apply a random transformation to the gamma of a tensor image.
+
+    This implementation aligns PIL. Hence, the output is close to TorchVision.
```

Comment:
is this confirmed or you just copy pasted from the others ?

Reply:
No I checked with torchvision.transform.adjustGamma. They produced the same result